Fastfile.private

```diff
@@ -206,19 +206,10 @@ reflected in `Info.plist` during building."
 
     build_configuration_key = options[:build_configuration]
     build_configuration = get_build_configuration(build_configuration: build_configuration_key)
-    match_type = get_match_type(build_configuration: build_configuration_key)
 
-    desc "Bundle identifier from xCode project"
-    bundle_identifier = read_project_property(
-      build_configuration: build_configuration,
-      build_setting: 'PRODUCT_BUNDLE_IDENTIFIER'
-    )
-
-    desc "Download provisioning profiles"
-    match(
-      app_identifier: bundle_identifier,
-      type: match_type,
-      readonly: true
+    desc "Download provisioning profile and certificates if not present."
+    refresh_certificates(
```

Comment:
A small ruby question: Why in this file you use this with parenthesis and in Fastfile (not private) just as `refresh_certificates build_configuration: :production`?

Reply:
No strong reason.

I use "no parenthesis" notation when there's a single parameter and fits in a single line.

I've added a new issue to double check this in the whole project: https://github.com/Wolox/fastlane-mobile/issues/13